doc/TRAINING.rst

```diff
@@ -219,6 +219,11 @@ Note: the released models were trained with ``--n_hidden 2048``\ , so you need t
 
    Key cudnn_lstm/rnn/multi_rnn_cell/cell_0/cudnn_compatible_lstm_cell/bias/Adam not found in checkpoint
 
+UTF-8 mode
+^^^^^^^^^^
+
+DeepSpeech includes a UTF-8 operating mode which can be useful to model languages with very large alphabets, such as Chinese Mandarin. For details on how it works and how to use it, see :ref:`decoder-docs`.
```

Comment:
"Chinese Mandarin" is a spoken language and has many written forms with different alphabets.

"Simplified Chinese" is a written form of "Chinese Mandarin" and has a single "large" alphabet.

So, here I'd say "Mandarin written using Simplified Chinese" instead of "Chinese Mandarin"